content/en/tracing/setup_overview/setup/dotnet-core.md

```diff
@@ -20,7 +20,7 @@ further_reading:
     text: "Runtime metrics"
   - link: "/serverless/azure_app_services/"
```

Comment:
There's a way (that I don't remember right now) to fix this link and have the old one redirect to the new one.

Reply:
That's a good question for the docs team. I didn't want to affect links so I only changed titles / content

Reply:
I found an example in our own `/content/en/tracing/setup_overview/setup/dotnet-core.md`. See the `aliases` section:

```
---
title: Tracing .NET Core Applications
kind: documentation
aliases:
  - /tracing/dotnet-core
  - /tracing/languages/dotnet-core
  - /tracing/setup/dotnet-core
  - /agent/apm/dotnet-core/
  - /tracing/setup/dotnet-core
  - /tracing/setup_overview/dotnet-core
```

Reply:
In any case, I don't think it's a blocker for this PR. Can be fixed later.

Reply:
Unless you have a strong objection, I'll just keep the URL the same because it's consistent with the URL's elsewhere and, most importantly, the PR already updates the page bodies

Reply:
Fine! 😆